communities/massmesh/meshnode/files/etc/config/dhcp

```diff
@@ -11,8 +11,9 @@ config dnsmasq
 	option leasefile '/tmp/dhcp.leases'
 	option nonwildcard '1'
 	option localservice '1'
-	list server '200:c493:950a:d576:a413:faef:990b:fb76'
+	list server '8.8.8.8'
 	list server '1.1.1.1'
+	list server '198.98.49.91'
```

Comment:
I'm not familiar with this public nameserver; do we expect it to be as reliable/longlasting as the Google or Cloudflare ones?

Reply:
it's an OpenNIC in NYC. Lowest latency I found.

Reply:
cool thanks for clarifying!